examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3122,11 +3122,9 @@ class GLTFParser {
 
 			if ( ! texture ) return null;
 
-			// Materials sample aoMap from UV set 1 and other maps from UV set 0 - this can't be configured
-			// However, we will copy UV set 0 to UV set 1 on demand for aoMap
-			if ( mapDef.texCoord !== undefined && mapDef.texCoord != 0 && ! ( mapName === 'aoMap' && mapDef.texCoord == 1 ) ) {
+			if ( mapDef.texCoord !== undefined ) {
 
-				console.warn( 'THREE.GLTFLoader: Custom UV set ' + mapDef.texCoord + ' for texture ' + mapName + ' not yet supported.' );
+				texture.uvSet = mapDef.texCoord;
```

Comment:
We'll also potentially need to clone the texture here, if `mapDef.texCoord` is > 0. Probably best to do that conservatively, we can try to reduce cloning later if it's an issue.

Reply:
Done: https://github.com/mrdoob/three.js/pull/25721/commits/ef9cee5d5d58cafe68728aa6521517ac74d275b6